lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -41,6 +41,11 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const descriptionEl = this.dom.find('.lh-metric__description', tmpl);
     descriptionEl.appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description));
 
+    if (audit.result.details) {
```

Comment:
convention would be adding something like `// HACK!` to call out this is a temporary thing ;)